app/src/main/java/com/nearbynest/app/utils/Urls.kt

```diff
@@ -19,7 +19,7 @@ package com.nearbynest.app.utils
  * API URLs collection.
  */
 object Urls {
-    private const val BASE_DEV = "https://randomuser.me/api/"
+    private const val BASE_DEV = "https://pixabay.com"
```

Comment:
You already created a new image service. Existing API service still exists. So both points to the same `BASE_URL` ? 

Then question arises why do you need new image service?